src/main/java/org/wso2/maven/humantask/artifact/HumanTaskMojo.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Used license config is obsolete. Please fix this and all other similar instances.